### ISSUE TITLE
Invalid test case TestHandler_SetupPasswordFile/SetupPasswordFile_no_mosquitto

### ISSUE THREAD

**Issue by bnevis-i** [edgexfoundry/edgex-go#4467]
# 🐞 Bug Report

### Affected Services [**REQUIRED**]
security bootstrapper (mosquitto)

### Is this a regression?
No

### Description and Minimal Reproduction [**REQUIRED**]
Issue is reproducible when running "make test" with the "mosquitto" snap installed.

## 🔥 Exception or Error
<pre><code>
--- FAIL: TestHandler_SetupPasswordFile (0.13s)
    --- FAIL: TestHandler_SetupPasswordFile/SetupPasswordFile_no_mosquitto (0.13s)
        handlers_test.go:206: Handler.GetCredentials() = true, want false
FAIL
</code></pre>

Root cause appears to be TestHandler_SetupPasswordFile calling SetupMosquittoPasswordFile
and SetupMosquittoPasswordFile running

```go
	cmd := exec.Command("mosquitto_passwd", "-c", "-b", pwdFile, handler.credentials.Username, handler.credentials.Password)
	_, err := cmd.Output()
```

In other words, this test case normally passes because mosquitto_passwd is not normally installed, and in this case, the command exists and does not terminate with an error.

This test case, if written properly would never try to *actually* exec mosquitto_passwd

## 🌍  Your Environment
**Deployment Environment:** Ubuntu 22.04
**EdgeX Version [**REQUIRED**]:** main
**Anything else relevant?** "mosquitto" snap is installed
